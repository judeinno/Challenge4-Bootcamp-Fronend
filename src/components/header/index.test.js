import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

it('testing Header', () => {
  const HeaderComponent = shallow(
    <Header />,
  );
  expect(HeaderComponent).toMatchSnapshot();
});
