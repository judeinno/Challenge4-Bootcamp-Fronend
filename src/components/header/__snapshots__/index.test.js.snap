// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Header 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light nav"
  >
    <a
      className="navbar-brand logo"
      href="/"
    >
      Stark Over Flow Lite
    </a>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav mr-auto"
      >
        <li
          className="nav-item active"
        >
          <a
            className="nav-link"
            href="/"
          >
            Home
            <span
              className="sr-only"
            >
              (current)
            </span>
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/login"
          >
            Login
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/signup"
          >
            Sign Up
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
